c/rust/Makefile

```diff
@@ -14,6 +16,14 @@ build:
 	    ;; \
```

Comment:
You need to remove the line

```
ar d target/debug/.libs/libvirtblocks_c_rust.a clzsi2.o;
```

because that file is no longer created for you now.